index.rst

```diff
@@ -0,0 +1,70 @@
+.. BharathSim-IISERPune documentation master file, created by
+   sphinx-quickstart on Sun Aug 15 12:05:07 2021.
+   You can adapt this file completely to your liking, but it should at least
+   contain the root `toctree` directive.
+
+Other Compartmental Models
+================================================
+
+SEIR
+====
+
+This is a generalisation made on the basic SIR model to include the Exposed state of the individuals. The exposed compartment (E) represents incubation period for the disease, that is a latent phase where the individual is infected but not yet infectious. The infected people can infect the susceptible (S) people who will be moved to the exposed (E) compartment before they are moved to the infected (I) compartment. From the infected compartment they will be removed (R) eventually.  The diagram below shows how the individuals move through each compartment in this model.
+
+.. image:: /seir-compartments.png
+
+The rate of transmission of the disease from an infected to a susceptible is represented by α. The incubation rate, β, is the rate of latent individuals becoming infectious. The average time an individual spends in the exposed compartment, the incubation period of the disease is thus given by 1/β. At last γ represents the rate of removal of infected individuals from Infected compartment.
```

Comment:
Also, let's not use new notation. Call the rate from S -> E \beta, and the rates out of E and I to be \lambda_E and \lambda_I respectively. Otherwise we'll soon run out of Greek alphabets, and it'll be confusing to compare different models.